src/main/java/ua/lviv/iot/novaPoshtaAPI/datastorage/dal/CourierFileStore.java

```diff
@@ -0,0 +1,181 @@
+package ua.lviv.iot.novaPoshtaAPI.datastorage.dal;
+
+import org.springframework.stereotype.Component;
+import ua.lviv.iot.novaPoshtaAPI.Util;
+import ua.lviv.iot.novaPoshtaAPI.model.Courier;
+
+import java.io.File;
+import java.io.OutputStreamWriter;
+import java.io.FileOutputStream;
+import java.io.Writer;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Component
+public class CourierFileStore {
+
+    public List<Courier> loadCouriers(boolean isTest) throws IOException {
+        List<Courier> resultList = new LinkedList<>();
+
+        File directory = new File("res");
+        if (!directory.exists()) {
+            directory.mkdir();
+        }
+
+        String testPath = "";
+        if (isTest) {
+            testPath = "test\\";
+        }
+        directory = new File("res\\test");
+        if (!directory.exists()) {
+            directory.mkdir();
+        }
+
+        File file;
+
+        String year = Integer.toString(LocalDate.now().getYear());
+        String month;
+
+        if (LocalDate.now().getMonthValue() < 10) {
+            month = "0" + LocalDate.now().getMonthValue();
+        } else {
+            month = Integer.toString(LocalDate.now().getMonthValue());
+        }
+
+        for (int i = 1; i <= LocalDate.now().getDayOfMonth(); i++) {
+            if (i < 10) {
+                if (Files.exists(Paths.get("res\\" + testPath + "courier-" + year + "-"
+                        + month + "-0" + i + ".csv"))) {
+                    file = new File("res\\" + testPath + "courier-" + year + "-"
+                            + month + "-0" + i + ".csv");
+                    resultList.addAll(scanCourier(file));
+                }
+            } else {
+                if (Files.exists(Paths.get("res\\" + testPath + "courier-" + year + "-"
+                        + month + "-" + i + ".csv"))) {
+                    file = new File("res\\" + testPath + "courier-" + year + "-"
+                            + month + "-" + i + ".csv");
+                    resultList.addAll(scanCourier(file));
```

Comment:
please use somehow src/main/java/ua/lviv/iot/novaPoshtaAPI/Util.java logic here

Reply:
it is used in saving logic. Loading one is different(we don't use "-day" here, from Util getDateNow)